modules/phase_field/test/tests/ad_diffusion_rate/gold/diffusion_rate.csv

```diff
@@ -0,0 +1,1002 @@
+time,mid_u
+0,0
+0.1,0.51011411334928
```

Comment:
I was wondering if we should use an exodus file instead of this huge CSV file (1K lines).

@aeslaughter Do we have guides on how large `csv` files we can make into repo?

Reply:
In terms of version control, I think plain text is better than a binary, even if it's many lines